cumulus/parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/bridge_to_ethereum_config.rs

```diff
@@ -24,7 +24,7 @@ use crate::{
 use parachains_common::{AccountId, Balance};
 use snowbridge_beacon_primitives::{Fork, ForkVersions};
 use snowbridge_core::{gwei, meth, AllowSiblingsOnly, PricingParameters, Rewards};
-use snowbridge_router_primitives::{inbound::MessageToXcm, outbound::EthereumBlobExporter};
+use snowbridge_router_primitives::{inbound::v1::MessageToXcm, outbound::EthereumBlobExporter};
```

Comment:
In https://github.com/paritytech/polkadot-sdk/pull/6706  I've moved outbound(router) primitives to separated crates,  do we want to apply the same pattern for inbound primitives?